ores/utilities/stress_test.py

```diff
@@ -127,7 +127,7 @@ def request_and_stat(ores_url, context, models, rev_ids, stats, verbose):
     try:
         response = requests.get(
             ores_url + path, params=params)
-    except:
+    except Exception:
```

Comment:
That is.... bending the rules.  This would be a good place to catch the more specific exception, IMO.

Reply:
This is identical behavior while addressing the flake8 error.  Seems like improving the error handling in stress_test.py should be left for future work.  In this case, I don't think it matters much given that it's just a test.